src/main/java/Controllers/TelevisionController.java

```diff
@@ -0,0 +1,59 @@
+package Controllers;
+
+import Exceptions.RecordNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+
+@RestController
+public class TelevisionController {
+
+    @GetMapping("television")
+        public ResponseEntity<String> getAllTelevisions(){
+            return  new ResponseEntity <> ("television", HttpStatus.OK);
+        }
+
+    @GetMapping("television/{id}")
+    public ResponseEntity<String> getOneTv (@PathVariable int id){
+        if (id == 5){
+            throw new RecordNotFoundException("een message");
```

Comment:
Dit werkt prima, maar "een message" is niet betekenisvol. Je kunt dan beter iets als "Het id is 5" meegeven. Dit is namelijk de tekst die de gebruiker uiteindelijk te zien krijgt (tenzij je natuurlijk een hardcoded tekst gebruikt, zoals je in de comment van de ExceptionController noemde)